### ISSUE TITLE
Description is now called definition and is inside of annotation (MI ontology)

### ISSUE THREAD

**Issue by tschaka1904** [EBISPOT/OLS#94]
Dear OLS-Team, 

as already mentioned to @simonjupp, we have problems to find the description of the MI terms. It seems like that it has moved into `annotation` and is called `definition`. @simonjupp said that reverting is changes would bring it back to normal, but it still seems the same.

We haven't made any changes to the [MI obo-file](https://github.com/MICommunity/psidev/blob/master/psi/mi/rel25/data/psi-mi25.obo), which could have caused this problem.

Best regards,
Max

**Comment by simonjupp** [comment]
Nothing has changed in the API, this was caused by the conversion from OBO to OWL when we changed to use your OBO file directly from github rather than using the OWL file generated by OBO library. This highlighted a problem with doing the conversion directly OWL API that we have now resolved. 

The issue with the OBO library has also been resolved and a new MI OWL file is available. We are reloading this now and everything will be back as it was, so you won't need to change anything in your code. I'll let you know when it is done. Same applies for #95
